<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-11-11 12:42:27.554 ***</b>
=ERROR REPORT==== 11-Nov-2024::12:42:27.554206 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244834.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-11-11 12:42:27.554 ***</b>
=CRASH REPORT==== 11-Nov-2024::12:42:27.554727 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244839.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244834.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244834.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-11-11 12:43:07.475 ***</b>
=ERROR REPORT==== 11-Nov-2024::12:43:07.475682 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.249185.0&gt;,
                               {#Ref&lt;0.3180401791.3915644929.249176&gt;,1006317,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.3180401791.3915644929.249176&gt;,1006317,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-11-11 12:43:07.476 ***</b>
=CRASH REPORT==== 11-Nov-2024::12:43:07.476109 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.249206.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3180401791.3915644929.249176&gt;,1006317,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.249185.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4509
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-11-11 12:47:44.875 ***</b>
=CRASH REPORT==== 11-Nov-2024::12:47:44.874876 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.255009.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3180401791.3917217793.52457&gt;,5773,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.255007.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 28663
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-11-11 12:47:53.827 ***</b>
=ERROR REPORT==== 11-Nov-2024::12:47:53.826767 ===
** Generic server <0.256293.0> terminating 
** Last message in was {nodedown,'peer_down_crash-378084-11742@8dbe664e4f0a'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-378084-11742",
                              peer_down => crash},
                            'peer_down_crash-378084-11742@8dbe664e4f0a',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,46,50,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-378084-11742",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDhkYmU2NjRlNGYwYQAD6SUAAAAAZzHrTA=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-378084-11742@8dbe664e4f0a'}},
                            {<0.256274.0>,#Ref<0.3180401791.3909615619.83291>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-378084-11742@8dbe664e4f0a'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-11-11 12:47:53.827 ***</b>
=CRASH REPORT==== 11-Nov-2024::12:47:53.827289 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256293.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-378084-11742@8dbe664e4f0a'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.256274.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 11232
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-11-11 12:47:54.904 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:47:53.902801 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-389317-11742@8dbe664e4f0a' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-11-11 12:47:55.372 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:47:55.372066 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-589442-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-11-11 12:47:55.416 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:47:55.416135 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-589378-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-11-11 12:48:10.975 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:09.973795 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-378340-11742@8dbe664e4f0a' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-11-11 12:48:11.311 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:11.311129 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-397027-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-11-11 12:48:11.393 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:11.393117 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-389701-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:13.576 ***</b>
=ERROR REPORT==== 11-Nov-2024::12:48:13.576617 ===
** Generic server <0.256652.0> terminating 
** Last message in was {'EXIT',<0.256625.0>,
                               {#Ref<0.3180401791.3917217793.70259>,1512101,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-397411-11742",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-397411-11742@8dbe664e4f0a',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,46,50,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-397411-11742",
                                     "-user","peer"],
                                    #Port<0.3121>,undefined,<<>>,running,
                                    {<0.256625.0>,
                                     #Ref<0.3180401791.3910139906.148543>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3180401791.3917217793.70259>,1512101,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:13.577 ***</b>
=CRASH REPORT==== 11-Nov-2024::12:48:13.577133 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256652.0>
    registered_name: []
    exception exit: {#Ref<0.3180401791.3917217793.70259>,1512101,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256625.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3121>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14159
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:14.212 ***</b>
=ERROR REPORT==== 11-Nov-2024::12:48:14.212594 ===
** Generic server <0.256680.0> terminating 
** Last message in was {'EXIT',<0.256630.0>,
                           {#Ref<0.3180401791.3917217793.70316>,2147921,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "WSTVCVBPITNMZDTSGSZQ","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@8dbe664e4f0a.2024-11-11_12.32.27/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-590722-11742",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@8dbe664e4f0a.2024-11-11_12.32.27/make_test_dir.stdlib_test.logs/run.2024-11-11_12.33.04/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-590722-11742",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.5/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-590722-11742@8dbe664e4f0a',
                                    "/usr/local/lib/erlang-26.2.5.5/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-590722-11742",
                                     "-setcookie","WSTVCVBPITNMZDTSGSZQ",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@8dbe664e4f0a.2024-11-11_12.32.27/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-590722-11742",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@8dbe664e4f0a.2024-11-11_12.32.27/make_test_dir.stdlib_test.logs/run.2024-11-11_12.33.04/log_private/",
                                     "-user","peer"],
                                    #Port<0.3149>,undefined,<<>>,running,
                                    {<0.256630.0>,
                                     #Ref<0.3180401791.3910139906.149143>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3180401791.3917217793.70316>,2147921,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:14.213 ***</b>
=CRASH REPORT==== 11-Nov-2024::12:48:14.213339 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256680.0>
    registered_name: []
    exception exit: {#Ref<0.3180401791.3917217793.70316>,2147921,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256630.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3149>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 10958
    stack_size: 29
    reductions: 20871
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:14.623 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:13.621757 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-390021-11742@8dbe664e4f0a' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:14.658 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:14.658146 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-590626-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:14.664 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:14.663995 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-390053-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-11-11 12:48:15.044 ***</b>
=WARNING REPORT==== 11-Nov-2024::12:48:15.044090 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-590754-11742@8dbe664e4f0a'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-11-11 13:10:09.096 ***</b>
=ERROR REPORT==== 11-Nov-2024::13:10:09.096331 ===
** Generic server <0.333574.0> terminating 
** Last message in was {'EXIT',<0.333571.0>,
                               {#Ref<0.3180401791.3947626497.141465>,351104,
                                [{child1,<0.333580.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.333574.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.333580.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458491],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3180401791.3947626497.141465>,351104,
    [{child1,<0.333580.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-11-11 13:10:09.096 ***</b>
=ERROR REPORT==== 11-Nov-2024::13:10:09.096342 ===
** Generic server <0.333573.0> terminating 
** Last message in was {'EXIT',<0.333571.0>,
                               {#Ref<0.3180401791.3947626497.141465>,351104,
                                [{child1,<0.333580.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.333573.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.333579.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458491],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3180401791.3947626497.141465>,351104,
    [{child1,<0.333580.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-11-11 13:10:09.097 ***</b>
=CRASH REPORT==== 11-Nov-2024::13:10:09.096745 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.333574.0>
    registered_name: []
    exception exit: {#Ref<0.3180401791.3947626497.141465>,351104,
                     [{child1,<0.333580.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.333571.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14912
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-11-11 13:10:09.097 ***</b>
=CRASH REPORT==== 11-Nov-2024::13:10:09.097053 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.333573.0>
    registered_name: []
    exception exit: {#Ref<0.3180401791.3947626497.141465>,351104,
                     [{child1,<0.333580.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.333571.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 15000
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-11-11 13:10:52.697 ***</b>
=CRASH REPORT==== 11-Nov-2024::13:10:52.697432 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.433743.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [bridge9212,<0.433738.0>]
    message_queue_len: 0
    messages: []
    links: [<0.433744.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4058
  neighbours:
    neighbour:
      pid: <0.433744.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.433743.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2024-11-11 13:37:30.428 ***</b>
=WARNING REPORT==== 11-Nov-2024::13:37:30.425805 ===
Process <0.437195.0> on node 'test_server@8dbe664e4f0a' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Mon Nov 11 2024 12:33:04<!--/date--><br />
</div>
</center>
</body>
</html>
